src/ui/units/ql/store/utils/grid.ts

```diff
@@ -105,7 +105,11 @@ export class Helper {
                 entryId: '',
                 type: '',
             },
-            version: QlConfigVersions.V2,
+            colors: [],
+            shapes: [],
+            labels: [],
+            tooltips: [],
```

Comment:
Can there be tooltips in QL too?

Reply:
Currently no, but I think we should have them for compatibility and probably we will have to add support of geovisualizations and tooltips in future